criteria/document-objectives.md

```diff
@@ -7,22 +7,26 @@ order: 8
 ## Requirements
 
 * The codebase MUST contain documentation of its objectives – like a mission and goal statement – that is understandable by developers and designers so that they can use or contribute to the codebase.
+* The components out of which the codebase SHOULD be composed MUST be documented by their functional isolation.
 * Codebase documentation SHOULD clearly describe the connections between policy objectives and codebase objectives.
 * The codebase MAY contain documentation of its objectives for the general public.
 
 ## Why this is important
 
 Documenting codebase objectives:
 
-* provides an easy way for people to decide whether this codebase is interesting for them now or in the future.
-* helps scope your own development.
-* clearly communicates to other stakeholders and contributors what the codebase is for.
+* provides an easy way for people to decide whether this codebase, or one of the components, is interesting for them now or in the future.
+* helps scope and isolate your own development.
+* improves maintanability and reusability.
+* channels (the improvements on the existing) functionality into logically isolated chunks.
```

Comment:
The gist of this is good, but the parenthesis makes the sentence a bit complex to parse. Could this be rewritten in more plain English somehow? It is okay using several sentences instead of one if it helps.

Reply:
As a non-native english speaker it's hard to come up with a better wording. Maybe @ericherman can suggest something?

Reply:
Perhaps "channels new functionality, and improvements to existing functionality, into logically isolated chunks."?

Reply:
If removing the parentheses makes all the difference, than just remove then.